samples/dotnet/kernel-syntax-examples/Example49_Apim.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.OpenAI;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using RepoUtils;
+
+// ReSharper disable once InconsistentNaming
+public static class Example49_Apim
+{
+    public static async Task RunAsync()
+    {
+        // Azute API Management details
+        // For more information see 'Protect your Azure OpenAI API keys with Azure API Management' here: https://learn.microsoft.com/en-us/semantic-kernel/deploy/
+        var apimUri = new Uri("https://apim...azure-api.net/");
```

Comment:
nit: Use environment variables like other examples?

Reply:
I'm seeing lot's of inconsistencies across the samples. I think @RogerBarreto has something planned to clean up the samples and having consistent configuration should be part of that.